nextflow.config

```diff
@@ -291,47 +291,53 @@ dag {
 
 manifest {
     name            = 'nf-core/scrnaseq'
-    author          = """Bailey PJ, Botvinnik O, Marques de Almeida F, Peltzer A, Sturm G""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
     contributors    = [
-        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
         [
             name: 'Bailey PJ',
             affiliation: '',
             email: '',
             github: '',
-            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
+            contribution: ['author'], // List of contribution types ('author', 'maintainer' or 'contributor')
             orcid: ''
         ],
         [
             name: ' Botvinnik O',
             affiliation: '',
             email: '',
-            github: '',
-            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
+            github: 'olgabot',
+            contribution: ['author'], // List of contribution types ('author', 'maintainer' or 'contributor')
             orcid: ''
         ],
         [
             name: ' Marques de Almeida F',
             affiliation: '',
             email: '',
-            github: '',
-            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
+            github: 'fmalmeida',
+            contribution: ['author'], // List of contribution types ('author', 'maintainer' or 'contributor')
             orcid: ''
         ],
         [
             name: ' Peltzer A',
             affiliation: '',
             email: '',
-            github: '',
-            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
+            github: 'apeltzer',
+            contribution: ['author'], // List of contribution types ('author', 'maintainer' or 'contributor')
             orcid: ''
         ],
         [
             name: ' Sturm G',
             affiliation: '',
             email: '',
-            github: '',
-            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
+            github: 'grst',
+            contribution: ['author'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: ' He D',
+            affiliation: '',
+            email: '',
+            github: 'DongzeHE',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
```

Comment:
Given that I’ve contributed extensively to the pipeline for this new release, if my contributions meet the criteria, I would really appreciate the opportunity to be listed as an 'author'. 

Of course, I completely understand if that’s not possible, and I appreciate the chance to be involved in the project."

Reply:
I would say it would make sense 😄 . However I am not sure if there are criteria a set of criterias for this, since I am not sure exaclty when I was added as well.

Will just loop-in @apeltzer and @grst . Do you have insights on it? If fine, then I can easily do the change.

😄

Reply:
I don't mind you being listed as "author". You are the main responsible for the simpleaf workflow, that should be more than enough.